test/helpers/data/workload-capacity-helper.js

```diff
@@ -0,0 +1,132 @@
+const config = require('../../../knexfile')
+const knex = require('knex')(config)
+var Promise = require('bluebird').Promise
+
+module.exports.addWorkloadCapacitiesForOffenderManager = function () {
+  var inserts = []
+
+  return knex('offender_manager_type').returning('id').insert({description: 'test'})
```

Comment:
personal pref, but what about having returning at the end as it'd be the last action of the call chain.